en/computer_vision/visual_inertial_odometry.md

```diff
@@ -0,0 +1,56 @@
+# Visual Intertial Odometry (VIO)
+
+*Visual Inertial Odometry* (VIO) is a [computer vision](../computer_vision/README.md) technique used for estimating the 3D *pose* (local position and orientation) and *velocity* of a moving vehicle relative to a *local* starting position.
+It is commonly used to navigate a vehicle in situations where GPS is absent or unreliable (e.g. indoors, or when flying under a bridge).
+
+VIO uses [Visual Odometry](https://en.wikipedia.org/wiki/Visual_odometry) to estimate vehicle *pose* from camera images, combined with inertial measurements from the vehicle IMU (to correct for errors associated with rapid vehicle movement resulting in poor image capture).
+
+<!-- video here -->
+
+
+
+## Supported Sensors 
+
+- [T265 Intel Realsense Tracking Camera](../peripherals/camera_t265_vio.md)
+
+
+## Setup Instructions
+
+You will need to attach your camera to the companion computer and set up ROS.
+Generally the camera should be mounted facing down. 
+If using the [T265 ](../peripherals/camera_t265_vio.md) it should also be soft-mounted.
```

Comment:
@baumanta 
1. What does soft mounting mean, and how do you do it with this camera?
2. How is this connected? Is it just a USB port?
3. How does ROS know what camera it is talking to? I.e. it seems magic to me that you can just attach this camera and somehow ROS know that it is what should be used.

Reply:
1. Kamil used some kind of "suckers". Like little springy things to attach the camera to frame in order to add some damping. The camera does not like vibrations.
2. yes as far as I remember a USB C. But I don't have a camera handy right now.
3. So from intel there is a package with the ROS wrapper for their cameras. You will need that and you need to strat the camera from ROS using the correct launch file

Reply:
Thanks!